sdlib/d/gc/bin.d

```diff
@@ -24,10 +22,12 @@ struct Bin {
 
 	void* alloc(shared(Arena)* arena, shared(ExtentMap)* emap,
 	            ubyte sizeClass) shared {
+		import d.gc.sizeclass;
```

Comment:
How come this import is necessary now? (or maybe, why wasn't it required before?).